libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/LoginActivity.kt

```diff
@@ -254,14 +254,13 @@ open class LoginActivity : AppCompatActivity(), OAuthWebviewHelperEvents {
         // If the intent is a callback from Chrome, process it and do nothing else
         if (isChromeCallback(intent)) {
             completeAuthFlow(intent)
+            webviewHelper?.clearView()
```

Comment:
This was a small ask from Salesforce App.  Because we make the identity call after auth is complete there is a gap of time where the custom tab has been closed but we don't dismiss the login activity yet.  This change means the user sees a white screen instead of a login screen.  Perhpas the spinner should be shown here a well?